### PR TITLE
Document DUPLICATE_USE_INSTANCING #2770

### PR DESCRIPTION
**Pull request by alexchond**

### CODE REVIEW

File: doc/classes/Node.xml [hunk 1]
```diff
@@ -994,6 +994,7 @@
 		</constant>
 		<constant name="DUPLICATE_USE_INSTANCING" value="8" enum="DuplicateFlags">
 			Duplicate using instancing.
+			(An instance stays linked to the original so when the original changes, 			the instance changes too.)
```
Comment (akien-mga) [inline doc/classes/Node.xml]: There's a lot of whitespace between the command and "the" which should be removed.
The comment is good, but I think the parentheses are not necessary.

**Comment by akien-mga** [comment]
I guess you meant to reference godotengine/godot-docs#2770 instead of #2770?

File: doc/classes/Node.xml (hunk 1 above)
Comment (alexchond) [reply doc/classes/Node.xml]: Yeah sorry I meant to reference godotengine/godot-docs#2770.
Should I create a new pull request with removed whitespace and parenthesis?
Sorry if that question sounded silly I'm kinda new to open source.

Comment (akien-mga) [reply doc/classes/Node.xml]: No need to make a new pull request, you can modify this one following this workflow: http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request

Comment (alexchond) [reply doc/classes/Node.xml]: done

**Comment by akien-mga** [comment]
Oops, clicked merge before seeing that there were two commits instead of one... Next time, please amend the original commit or rebase to squash them instead of making several commits for a single change (see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) for Git usage instructions matching this workflow).

Thanks nevertheless for the docs improvement :)

**Comment by alexchond** [comment]
Sorry didn't know about that.
I'll read it carefully and hope my next commit is without any faults!
Thanks for this incredible open source project.
I'll try to contribute more. :)

**Pull request merged**